include/blobify/stream_storage.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef BLOBIFY_STREAM_STORAGE_HPP
 #define BLOBIFY_STREAM_STORAGE_HPP
 
+#include <iostream>
```

Comment:
Could you swap this out for `iosfwd` to go a bit softer on compile times?

Reply:
I tried, but line 16 calls std::istream::ignore so gcc13 and clang17 both complain it is `invalid use of incomplete type ‘std::istream’` or `implicit instantiation of undefined template`

Reply:
Ah, right. Thanks for checking though!